integration/ct_integration_test.go

```diff
@@ -95,3 +109,18 @@ func ctTimestampToTime(ts uint64) time.Time {
 func signatureToString(signed *ct.DigitallySigned) string {
 	return fmt.Sprintf("Signature: Hash=%v Sign=%v Value=%x", signed.Algorithm.Hash, signed.Algorithm.Signature, signed.Signature)
 }
+
+func certsFromPEM(data []byte) []ct.ASN1Cert {
```

Comment:
Thinking more, `LoadPrivateKey` is kind of silly -- I can re-use `crypto/key_manager.go` directly.